### PR TITLE
Add composed mode to actions.

### PR DESCRIPTION
**Pull request by tcooper-uk**
Adds the OASDiff composed mode to the actions.

### CODE REVIEW

**Review by effoeffi — COMMENTED** [review]
Thanks for your contribution.
Can you please add a test and this option to readme?

**Comment by tcooper-uk** [comment]
> Thanks for your contribution. Can you please add a test and this option to readme?

Sure, all done now @effoeffi.

File: README.md [hunk 1]
```diff
@@ -12,8 +12,8 @@ Copy and paste the following snippet into your build .yml file:
 - name: Running OpenAPI Spec diff action
   uses: oasdiff/oasdiff-action/diff@main
   with:
-    base: 'specs/base.yaml'
-    revision: 'specs/revision.yaml'
+    base: 'specs/base/base.yaml'
+    revision: 'specs/revision/revision.yaml'
```
Comment (effoeffi) [inline README.md:16]: Have you moved base and revision yamls?

Comment (tcooper-uk) [reply README.md]: Appologies, I was not supposed to commit that, fixed.

**Pull request merged**